src/20_Components/amp-analytics.html

```diff
@@ -19,12 +20,16 @@
   <title>amp-analytics</title>
   <script async src="https://cdn.ampproject.org/v0.js"></script>
   <!-- ## Setup -->
-  <!--  Import the amp-analytics component in the header. -->
+  <!--  Import the `amp-analytics` component in the header. -->
   <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
-  <!--  Import the amp-carousel component in the header. -->
+  <!--  Import the `amp-carousel` component in the header. -->
```

Comment:
I think it's fine to remove the documentation from all non analytics related extensions as I don't think they need to be listed on the doc page. Wdyt?

Reply:
done